docker/marathon.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 
 lookup_host() {
-    nslookup "$1" | awk -v HOST="$1" '{ if ($2 == HOST) { getline; gsub(/^.*: /, ""); print; } }'
+    nslookup "$1" | awk -v HOST="$1" '{ if ($2 == HOST) { getline; gsub(/^.*: /, ""); split($0, a, " ", seps); print a[1]; } }'
```

Comment:
`print $1` should do the same thing. Would you mind testing if that works for you?